framework/src/base/NonlinearSystemBase.C

```diff
@@ -1617,9 +1623,7 @@ NonlinearSystemBase::constraintJacobians(SparseMatrix<Number> & jacobian, bool d
 #endif
 #endif
 
-        jacobian.close();
         jacobian.zero_rows(zero_rows, 0.0);
-        jacobian.close();
```

Comment:
I'm happy to see all these extra(?) closes go, but are we sure they aren't needed? Perhaps @friedmud will need to chime in.